src/Logging/Logging.Debug/DebugLogService.cs

```diff
@@ -1,45 +1,25 @@
-﻿using static CodeMonkeys.Argument;
-
-using System;
-
-namespace CodeMonkeys.Logging.Debug
+﻿namespace CodeMonkeys.Logging.Debug
 {
-    internal sealed class DebugLogService : ILogService
+    /// <summary>
+    /// <see cref="IScopedLogService"/> which writes to <see cref="System.Diagnostics.Debug"/>.
+    /// </summary>
+    public sealed class DebugLogService : ScopedLogService<DebugLogOptions>
     {
-        private readonly DebugLogServiceProvider _provider;
-        private readonly string _context;
-
-        internal DebugLogService(
-            DebugLogServiceProvider provider, 
-            string context)
+        internal DebugLogService(string context)
+            : base(context)
         {
-            _provider = provider;
-            _context = context;
         }
 
-        public bool IsEnabledFor(LogLevel logLevel) => _provider.IsEnabledFor(logLevel);
-
-        public void Log<TState>(
-            DateTimeOffset timestamp, 
-            LogLevel logLevel, 
-            TState state, 
-            Exception ex, 
-            Func<TState, Exception, string> formatter)
+        protected override void PublishMessage(LogMessage message)
         {
-            NotNull(formatter, nameof(formatter));
-
-            _provider.ProcessMessage(new LogMessage(
-                timestamp,
-                logLevel,
-                formatter(state, ex),
-                _context,
-                ex));
+            try
+            {
+                System.Diagnostics.Debug.WriteLine(
+                        MessageFormatter.Format(
+                            message,
+                            Options.TimeStampFormat));
+            }
+            catch { }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Handle the exception or explain in a comment why it can be ignored.](https://app.codacy.com/gh/UltimateCodeMonkeys/CodeMonkeysMVVM/pullRequest?prid=5626772)